docs/azure_reference.md

```diff
@@ -71,7 +71,11 @@ Now authenticate:
 
 ### Remote Console 
 
-To use a remote system, after connecting to the Azure environment with an SSL tunnel or by using a VPN or ExpressRoute, run the configuration step `az login` before using vfxt.py.
+For Azure-hosted virtual machines, you can use **az login --identity** to obtain a token that allows you to use the `from-environment` option to log in. 
```

Comment:
This is actually for any machine with the az command line, not just Azure-hosted VMs.  Azure hosted VMs with managed identity setup can use --on-instance for auth instead of setting up az login and --from-environment.